crates/cards/tests/basic.rs

```diff
@@ -0,0 +1,27 @@
+#![allow(dead_code, clippy::unwrap_used, clippy::panic_in_result_fn)]
+
+use cards::CardSecurityCode;
+/// use cards::CardExpirationMonth;
+/// use cards::CardExpirationYear;
+use cards::CardExpiration;
+
+use masking::PeekInterface;
+use masking::StrongSecret;
+
+#[test]
+fn basic() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    
+    let csc =  StrongSecret::<u16>::new(1234);
+    let expiry_month = StrongSecret::<u8>::new(12);
+    let expiry_year = StrongSecret::<u16>::new(2023);
+    
+
+    let card_security_code = CardSecurityCode::new(csc).unwrap();
+    let card_expiration = CardExpiration::new(expiry_month, expiry_year).unwrap();
+
+    assert_eq!(*card_security_code.peek().peek(), 1234);
+    assert_eq!(*card_expiration.get_month().peek().peek(), 12);
+    assert_eq!(*card_expiration.get_year().peek().peek(), 2023);
+
+    Ok(())
+}
```

Comment:
This is an integration test. I'd suggest using unit tests in the crate using `#[cfg(test)]`. Also, add test cases for validations for assurance that the code will be panic free at runtime, eg:
- 31st April 20xx (error, April has only 30 days)
- 31st Feb 20xx (error, Feb has max 28/29 days)
- 29th Feb 2020 (Valid leap year)
- 29th Feb 2021 (error, 2021 is not a leap year)

Add more as you see fit.

Reply:
Ignore this, I just saw that the `time` crate handles all of these cases for you! :)